Cargo.lock

```diff
@@ -2906,7 +2906,7 @@ dependencies = [
 [[package]]
 name = "zstd-sys"
 version = "1.4.10+zstd.1.4.0"
-source = "git+https://github.com/gyscos/zstd-rs.git#d09a3a06e2802e92f011bb362fa2b4c264ed6f41"
+source = "git+https://github.com/gyscos/zstd-rs.git#f5d0cddc58a1b164e7312164465d11bc701af83e"
```

Comment:
Is this change required?

Reply:
I don't think so. But this is done by `cargo update -p rocksdb`.